editoast/src/views/paced_train.rs

```diff
@@ -137,6 +137,43 @@ async fn get_by_id(
     Ok(Json(paced_train))
 }
 
+/// Update a paced train
+#[utoipa::path(
+    put, path = "",
+    tag = "timetable,paced_train",
+    params(PacedTrainIdParam),
+    request_body = inline(PacedTrainBase),
+    responses(
+        (status = 200, body = PacedTrainResult, description = "Paced train have been updated")
```

Comment:
```suggestion
        (status = 204, description = "The paced train has been updated")
```

no need to return the request body unchanged

Reply:
Without `body = PacedTrainResult` we lose the type in openapi and TS types

Reply:
I'm suggesting to not return anything for this endpoint as it's a PUT and not a PATCH, so its request's payload won't differ from its response's